### PR TITLE
First content

### PR DESCRIPTION
**Pull request by odineiramone**
@darlanmendonca and @khaosdoctor, take a look, please!

Just a articles about installation...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+# Program is trivial in
+# Ruby. Superfluous:
+#
+# * A "main" method
+# * Newline
+# * Semicolons
+#
+# Here is the Code:
+
+puts "Hello World!"
```
Comment (khaosdoctor) [inline README.md]: I don't think this code block is actually needed...

File: README.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+
+# now, lets change the game!
+```
+
+_Ruby is a  programming language developed in 1995 by [Yukihiro "Matz" Matsumoto](https://github.com/matz). was influenced by Python, Perl, Smalltalk, Eiffel, Ada e Lisp_ (That is history enough to get started)
+
+## Getting started
```
Comment (khaosdoctor) [inline README.md]: Will it be a topic structure? Or it is just the initial parts only?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+
+Let's start of the very beginning. How to install ruby?
+
+- First, choice your ~~destiny~~ operational system
+  - [Ubuntu](/contents/en/how_to_install_on_ubuntu.md)
+  - [MacOS](/contents/en/how_to_install_on_macos.md)
+  - [Windows](/contents/en/how_to_install_on_windows.md)
+
+*Olá! Aqui falamos português também!*<br>
```
Comment (khaosdoctor) [inline README.md]: I think this should be moved to an quote right on top of the document. The first thing a Brazilian user has to see is this...

However I'm not comfortable with the idea of creating 2 separated readme files

File: contents/en/404_en.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+![404](../../images/404.png)
+
+Sorry, the page you are looking for is still being written! Enquanto Meanwhile, take a look at our [README](../../README.md), has lots of cool stuff there!
```
Comment (khaosdoctor) [inline contents/en/404_en.md]: "Enquanto"

File: contents/en/how_to_install_on_windows.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# How to install Ruby on Windows
+
+Unlike [Ubuntu](como_instalar_no_ubuntu.md) and [MacOS](como_instalar_no_macos.md), install Ruby on Windows don't need a version manager, you'll just need a wizard just like any other Windows software
```
Comment (khaosdoctor) [inline contents/en/how_to_install_on_windows.md]: Filenames are in portuguese?

File: contents/pt_br/como_instalar_no_macos.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Como instalar o Ruby no MacOS
+
+## Installing Rbenv
+
+- É tãããããão fácil! Dê uma olhada em como fazer essa instalação no [Ubuntu](how_to_install_on_ubuntu.md) e compare!
```
Comment (khaosdoctor) [inline contents/pt_br/como_instalar_no_macos.md]: I think the file links are mistaken, the english files are pointing to portuguese ones and vice versa

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
There are some changes to be made, I think we'd better discuss the idea of creating two separate README files...

**Comment by odineiramone** [comment]
@khaosdoctor, I carried away. I remade the file structure and now it's more simple and easy to use and maintain

**Comment by odineiramone** [comment]
@khaosdoctor, I'll merge these changes. If you want request anything more, open an issue please :D

**Pull request merged**